liquibase-standard/src/main/java/liquibase/datatype/core/TimestampType.java

```diff
@@ -126,19 +126,17 @@ public DatabaseDataType toDatabaseDataType(Database database) {
             type = new DatabaseDataType("TIMESTAMP");
         }
 
-        if (originalDefinition.startsWith("java.sql.Types.TIMESTAMP_WITH_TIMEZONE")
+        if (originalDefinition.toUpperCase().startsWith("JAVA.SQL.TYPES.TIMESTAMP_WITH_TIMEZONE")
             && (database instanceof PostgresDatabase
             || database instanceof OracleDatabase
             || database instanceof H2Database
             || database instanceof HsqlDatabase
             || database instanceof SybaseASADatabase)) {
 
-            if (database instanceof PostgresDatabase
-            || database instanceof H2Database
-            || database instanceof SybaseASADatabase) {
-                type.addAdditionalInformation("WITH TIME ZONE");
-            } else {
+            if (database instanceof HsqlDatabase) {
```

Comment:
PR title says Oracle, but here we are changing other databases behavior. Is it expected?

Reply:
I changed the order of the logic. Previously, `Postgres`, `H2`, and `SybaseASADatabase` were already processing as `WITH TIME ZONE`, so I thought I could revert the logic of this IF, instead of adding `Oracle` to the existing IF. 

I can revert this and just add `Oracle` to the previous `IF` logic.